mmv1/third_party/terraform/services/container/resource_container_node_pool_test.go.erb

```diff
@@ -3016,19 +3026,35 @@ resource "google_container_node_pool" "with_pco_disabled" {
   location           = "us-central1"
   cluster            = google_container_cluster.cluster.name
   node_count = 1
+  node_config {
+	oauth_scopes = [
+	  "https://www.googleapis.com/auth/cloud-platform",
+	]
+  }
+}
+
+resource "google_container_node_pool" "with_tier1_net" {
+  name               = "%s-tier1"
+  location           = "us-central1"
+  cluster            = google_container_cluster.cluster.name
+  node_count = 1
   network_config {
-        pod_cidr_overprovision_config {
-		disabled = true
+	network_performance_config {
+		total_egress_bandwidth_tier = "%s"
 	}
   }
   node_config {
+	machine_type = "n2-standard-32"
```

Comment:
Oh, the issue is from the test alone- we need to use a smaller machine or different type:

```
Insufficient regional quota to satisfy request: resource "N2_CPUS": request requires '96.0' and is short '72.0'. project has a quota of '24.0' with '24.0' available.
```

Reply:
Per GKE docs [1] "Tier_1 networking is supported on N2, N2D, C2, C2D, C3, C3D, and M3 VMs that have at least 30 vCPUs.".

If the quota is insufficient for N2s, I doubt there will be sufficient quota for a different type out of those, is it right?  I think we can either skip this test entirely, bump the quota or make it manual-only in some way, what would you recommend?

[1] https://cloud.google.com/compute/docs/networking/configure-vm-with-high-bandwidth-configuration

Reply:
Could we use a zonal cluster or specify a single node location? That'll reduce us from 96 to 32 CPUs.

Reply:
Makes sense, changed to a single node location.